src/views/product-landing/server.ts

```diff
@@ -26,12 +36,48 @@ async function generateStaticProps({
   // TODO: content is accurate, while not needing to be
   // TODO: manually kept up to date?
   // Asana task: https://app.asana.com/0/1201010428539925/1201646299837754/f
+  const usedHeadings = []
   CONTENT.blocks = CONTENT.blocks.map((block) => {
-    const { type, heading } = block
-    if (type !== 'heading') return block
-    return {
-      ...block,
-      slug: slugify(heading, { lower: true }),
+    switch (block.type) {
+      case 'heading':
+        // augment heading blocks with a consistent slug,
+        // used for anchor linking
+        // if the resulting slug is not unique, append an
+        // integer suffix to make sure that it is
+        // eslint-disable-next-line no-case-declarations
+        const baseSlug = slugify(block.heading, { lower: true })
+        // eslint-disable-next-line no-case-declarations
+        let slug = baseSlug
+        // eslint-disable-next-line no-case-declarations
+        let slugMakeUniquePrefix = 0
+        while (usedHeadings.indexOf(slug) !== -1) {
+          slugMakeUniquePrefix++
+          slug = `${baseSlug}-${slugMakeUniquePrefix}`
+        }
+        usedHeadings.push(slug)
+        return { ...block, slug }
```

Comment:
At least for now, we have duplicate `Lorem ipsum` headings, this is how this came up!

For production content, I think there's a pretty strong chance we may have duplicate headings - maybe less likely than on a typical MDX docs or learn page (those are almost certain to have duplicate headings in some documents), but figured it was likely enough to make it worth proactively accounting for 👍

Let me know if that helps! Open to other approaches here in subsequent PRs, just didn't want the jump links for any of the product pages to seem broken with the `Lorem ipsum` in place.

Reply:
That makes sense! I don't think we want to allow duplicate headings since they are landmarks and should be unique, but that's a conversation outside of this PR. 😊